RPRMakie/examples/volume.jl

```diff
@@ -1,16 +1,44 @@
-using RPRMakie
+using RPRMakie, ProgressMeter
 using Makie, NIfTI, FileIO
 using GLMakie
 r = LinRange(-1, 1, 100)
```

Comment:
```suggestion
r = range(-1, 1, 100)
```

Reply:
Rather be expressive about range IMO, will probably keep this as LinRange for now and maybe switch later.